pheps/phep-9999.md

```diff
@@ -0,0 +1,112 @@
+| **Gold**              | Completed              | Mostly green, some yellow allowed | Completed               | No conflicts allowed            | Required     | Interoperable with all other PyHC core packages | Yes              | Yes                |
+| **Bronze**            | Completed              | Several yellow, no red            | In Progress             | A couple conflicts exist        | Required     | Interoperable with most PyHC core packages      | Yes              | No                 |
+| **Silver**            | Completed              | Red grades allowed                | Not Completed           | Major conflicts exist           | Required     | Interoperable with 1-2 PyHC core packages       | Yes              | No                 |
+| **Honorable Mention** | Not Done               | N/A                               | N/A                     | Major conflicts exist           | Not required | Does not interoperate with core packages        | No               | No                 |
+
+Descriptions for each heading are as follows:
+   - Self Evaluation Status: indicates whether a package has completed a self evaluation against PyHC's standards
+   - PyHC Standard Grades: indicates status of each standards grouping within a package's self evaluation
+   - pyOpenSci Review Status: indicates status of a pyOpenSci review
+   - PyHC Env Installation Conflicts: indicates state of installation conflicts within the PyHC software environment
+   - DOI: indicates whether or not a package has a DOI (e.g., from Zenodo or a publication)
+   - Interoperability Status: indicates the level of interoperability a package has with PyHC core packages
+   - pip Installable: indicates whether a package is pip installable
+   - conda Installable: indicates whether a package is conda installable
+
+The following table shows the benefits that are associated with each tier:
+
+| Tier                  | Summer School Inclusion | PyHC Software Env Inclusion | PyHC-Chat Bot Inclusion | pyOpenSci Verified badge | Standards Compliance Assistance | Listing on Main PyHC Project Page | Listing on Secondary PyHC Project Page | Software Search Interface Inclusion | Consideration for Conference Travel Funding |
+| :--:                  | :---------------------: | :-------------------------: | :---------------------: | :----------------------: | :-----------------------------: | :-------------------------------: | :------------------------------------: | :---------------------------------: | :-----------------------------------------: |
+| **Gold**              | Yes                     | Yes                         | Yes                     | Yes                      | Yes                             | Yes                               | No                                     | Yes                                 | Yes                                         |
```

Comment:
Considering that I don't think any packages would be at a gold level right now, that would suggest no packages at a figure summer school.

Reply:
True. Happy to modify that one.